rapids-cmake/cuda/detail/detect_architectures.cmake

```
@@ -49,7 +49,12 @@ int main(int argc, char** argv)
       char buff[32];
       cudaDeviceProp prop;
       if (cudaGetDeviceProperties(&prop, dev) != cudaSuccess) { continue; }
-      sprintf(buff, \"%d%d\", prop.major, prop.minor);
+      if(prop.major >= 9) {
```

I did some formatting work on this recently. ;)
```suggestion
      if (prop.major >= 9) {
```